redblueapp/Dockerfile

```diff
@@ -3,13 +3,19 @@ FROM drupal:8.5.0
 RUN apt-get update
 RUN apt-get -y install supervisor
 RUN apt-get -y install iputils-ping
+RUN apt-get -y install libssl-dev
 
 ADD supervisord.conf /etc/supervisor/conf.d/supervisord.conf 
 
 COPY drupal-8.5.0 /var/www/html
 RUN mkdir /opt/net-diag
 COPY net-diag /opt/net-diag
 COPY drupal-8.5.0 /var/www/html
+
+ENV MASTER_SALT ABCD
```

Comment:
We can leave this one out. The master salt will be passed in by ECS or by docker with -e

Reply:
Done